src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java

```diff
@@ -24,12 +24,14 @@
      * Type B is a class that extends {@link AbstractBuildingWorker.View}.
      */
     protected final B          building;
-    private final   SwitchView switchView;
     private final   Label      title;
```

Comment:
### Errors:
- Type of field 'title' is concrete class Label 

### Warnings:
- Instance field name title doesn't match regex 'm_[a-z][A-Za-z\d]*'